GoogleSignIn/Sources/GIDVerifyAccountDetail/Implementations/GIDVerifiableAccountDetail.m

```diff
@@ -37,4 +39,20 @@ - (nullable NSString *)scope {
   }
 }
 
+- (BOOL)isEqual:(id)object {
```

Comment:
You should implement `-hash` if you implement `-isEqual`. Further reading available in [the docs](https://developer.apple.com/documentation/objectivec/1418956-nsobject/1418795-isequal#discussion).

Reply:
Done.